python/sedona/spark/__init__.py

```diff
@@ -43,3 +43,7 @@
 from sedona.raster_utils.SedonaUtils import SedonaUtils
 from sedona.maps.SedonaKepler import SedonaKepler
 from sedona.maps.SedonaPyDeck import SedonaPyDeck
+from sedona.sql.st_aggregates import *
+from sedona.sql.st_constructors import *
+from sedona.sql.st_functions import *
```

Comment:
This introduces another problem: st_functions defines a function named `GeometryType`, it replaces the previously imported `from sedona.sql.types import GeometryType`.